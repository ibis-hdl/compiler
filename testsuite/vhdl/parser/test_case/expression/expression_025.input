-- don't interpret a comment '--' as neg. sign
(a * b - -- comment
 c)
