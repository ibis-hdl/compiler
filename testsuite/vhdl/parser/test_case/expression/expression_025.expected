(expression
  (relation
    (shift_expression
      (simple_expression
        (term
          (primary<v>
            (expression
              (relation
                (shift_expression
                  (simple_expression
                    (term
                      (primary<v>
                        (name<v>
                          (identifier
                            a
                          )
                        )
                      ),
                      (operator: MUL),
                      (primary<v>
                        (name<v>
                          (identifier
                            b
                          )
                        )
                      )
                    ),
                    (operator: SUB),
                    (term
                      (primary<v>
                        (name<v>
                          (identifier
                            c
                          )
                        )
                      )
                    )
                  )
                )
              )
            )
          )
        )
      )
    )
  )
)
